src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reverseRobot = {};
+
+  for (const key in robot) {
+    if (!reverseRobot.hasOwnProperty(robot[key])) {
+      reverseRobot[robot[key]] = key;
+    } else {
+      return null;
+    }
```

Comment:
Slight improvement - in general human better understands positive conditions, rather than negative

i.e. the following is slightly better readable

```suggestion
    if (reverseRobot.hasOwnProperty(robot[key])) {
      return null
    } 
    
    reverseRobot[robot[key]] = key;
```